docs/concepts/admission-webhook.md

```diff
@@ -0,0 +1,44 @@
+# Admission-webhook
+
+Lokomotive has an admission-webhook which runs in it's own namespace `lokomotive`. This document explains how the webhook works adding more security to your cluster. 
```

Comment:
```suggestion
As part of the cluster control-plane Lokomotive creates additional admission webhooks, which provides extra security features to the cluster. This document describes what webhooks we install and what they do.
```

Reply:
I'd link in this paragraph to explanation what admission webhooks do in K8s documentation.